app/javascript/stylesheets/application.scss

```diff
@@ -1,6 +1,25 @@
 @import "~bootstrap/scss/bootstrap";
 @import "./_sidebar";
+@import url('https://fonts.googleapis.com/css2?family=Lobster&display=swap');
 
 #page-content-wrapper {
   width: 100%;
 }
+
+.big-button {
+  width: 100%;
+}
+
+.column-background {
+  display: flex;
+  flex-direction: column;
+  object-fit: contain;
+  background-image: linear-gradient(103deg, #222E62, #6A98CC);
+  background-repeat: no-repeat;
+  background-size: cover;
+  align-items: center;
+  justify-content: center;
+  color: white;
+  height: 100vh;
+  font-family: 'Lobster', cursive;
```

Comment:
Is the font here different than for the rest of the website?

Reply:
yes, it's another font. I guess the rest of the website uses Roboto